cardano-node/src/Cardano/Tracing/Tracers.hs

```diff
@@ -273,417 +278,479 @@ mkTracers traceConf tracer bcCounters = do
+        , handshakeTracer = tracerOnOff (traceHandshake traceConf) "Handshake" tracer
+        , localHandshakeTracer = tracerOnOff (traceLocalHandshake traceConf) "LocalHandshake" tracer
+        }
+    TracingOff ->
+      pure Tracers
+        { chainDBTracer = nullTracer
+        , consensusTracers = mkConsensusTracers' elidedFetchDecision blockForgeOutcomeExtractor forgeTracers nullTracer TracingOff  bcCounters
+        , nodeToClientTracers = nodeToClientTracers' TracingOff nullTracer
+        , nodeToNodeTracers = nodeToNodeTracers' TracingOff nullTracer
+        , ipSubscriptionTracer = nullTracer
+        , dnsSubscriptionTracer= nullTracer
+        , dnsResolverTracer = nullTracer
+        , errorPolicyTracer = nullTracer
+        , localErrorPolicyTracer = nullTracer
+        , acceptPolicyTracer = nullTracer
+        , muxTracer = nullTracer
+        , handshakeTracer = nullTracer
+        , localHandshakeTracer = nullTracer
+        }
+
+--------------------------------------------------------------------------------
+-- Chain DB Tracers
+--------------------------------------------------------------------------------
+
+teeTraceChainTip
+  :: ( Condense (HeaderHash blk)
+     , LedgerSupportsProtocol blk
+     , ToObject (Header blk)
+     )
+  => TraceOptions
+  -> IORef BlockchainCounters
+  -> MVar (Maybe (WithSeverity (ChainDB.TraceEvent blk)), Integer)
+  -> Trace IO Text
+  -> Tracer IO (WithSeverity (ChainDB.TraceEvent blk))
+teeTraceChainTip tconf bChainCounters elided tr =
+  case tconf of
+    TracingOff -> nullTracer
+    TracingOn tSelect ->
+      Tracer $ \ev -> do
+       traceWith (teeTraceChainTip' tr) ev
+       traceWith (notifyForkIsCreated bChainCounters tr) ev
+       traceWith (teeTraceChainTipElide (traceVerbosity tSelect) elided tr) ev
+
+teeTraceChainTipElide
+  :: ( Condense (HeaderHash blk)
+     , LedgerSupportsProtocol blk
+     , ToObject (Header blk)
+     )
+  => TracingVerbosity
+  -> MVar (Maybe (WithSeverity (ChainDB.TraceEvent blk)), Integer)
+  -> Trace IO Text
+  -> Tracer IO (WithSeverity (ChainDB.TraceEvent blk))
+teeTraceChainTipElide = elideToLogObject
+
+traceChainInformation :: Trace IO Text -> ChainInformation -> IO ()
+traceChainInformation tr ChainInformation { slots, blocks, density } = do
+  -- TODO this is executed each time the chain changes. How cheap is it?
+  meta <- mkLOMeta Critical Confidential
+  let tr' = appendName "metrics" tr
+      traceD :: Text -> Double -> IO ()
+      traceD msg d = traceNamedObject tr' (meta, LogValue msg (PureD d))
+      traceI :: Integral a => Text -> a -> IO ()
+      traceI msg i = traceNamedObject tr' (meta, LogValue msg (PureI (fromIntegral i)))
+
+  traceD "density"     (fromRational density)
+  traceI "slotNum"     slots
+  traceI "blockNum"    blocks
+  traceI "slotInEpoch" (slots `rem` epochSlots)
+  traceI "epoch"       (slots `div` epochSlots)
+ where
+   epochSlots :: Word64 = 21600  -- TODO
```

Comment:
Please rebase after https://github.com/input-output-hk/cardano-node/pull/1108 is merged.